docs/configuration.rst

```diff
@@ -67,9 +67,11 @@ Cluster
 +------------------------------------------+------------------------------------------------------------------------------------------------+-------------------+-------------------------------------------------+-----------------+
 | Option                                   | Description                                                                                    | Type              | Default                                         | Client / Server |
 +==========================================+================================================================================================+===================+=================================================+=================+
```

Comment:
Fix spacing for the options to align the table, otherwise it will not render correctly
(This can be checked with the Rich Diff view for this file)